### PR TITLE
bug(SWA-228): use consistent error message format

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **bug**

[SWA-228]: use consistent error message format

**To reviewers :** For some reason there is a lot of white space diffs that you can turn off by selecting "Hide whitespace" 

Screenshot 2022-02-25 at 18 48 21
 
Screenshot 2022-02-25 at 14 08 35

This PR resolves []

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

ⓘ 'User-facing' changes will be published as release notes.

#### Cross-platform user-facing changes

- Rename modal title - patrinoua

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[SWA-228]: https://artsyproduct.atlassian.net/browse/SWA-228?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Rename modal title - patrinoua)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4bd99edfbfcab2dbe9081ab81de777cda1b09648
</p>

File: src/app/Components/PhoneInput/PhoneInput.tests.tsx [hunk 1]
```diff
@@ -114,8 +114,8 @@ describe("PhoneInput", () => {
     expect(tree.root.findAllByType(Text)[2].props.children).toBe("This phone number is incomplete")
   })
 
-  it("does not show a validation message when phone number valid", () => {
-    const tree = renderWithWrappers(
+  it.only("does not show a validation message when phone number valid", () => {
```
Comment (brainbicycle) [inline src/app/Components/PhoneInput/PhoneInput.tests.tsx:117]:
```suggestion
  it("does not show a validation message when phone number valid", () => {
```
it.only left by mistake?

File: src/palette/elements/Input/Input.tests.tsx [hunk 2]
```diff
@@ -33,4 +33,10 @@ describe("Input", () => {
 
     expect(value).toEqual("mockStr")
   })
+
+  it.only("Shows an error message when input has an error", () => {
```
Comment (brainbicycle) [inline src/palette/elements/Input/Input.tests.tsx:37]:
```suggestion
  it("Shows an error message when input has an error", () => {
```
it.only left by mistake?

**Review by brainbicycle — COMMENTED** [review]
This looks good to me! some of the tests still have `it.only` once that is fixed good to merge!

File: src/palette/elements/Input/Input.tests.tsx (hunk 2 above)
Comment (patrinoua) [reply src/palette/elements/Input/Input.tests.tsx:37]: yes @brainbicycle, thanks for catching that!

File: src/app/Components/PhoneInput/PhoneInput.tests.tsx [hunk 3]
```diff
@@ -115,7 +115,7 @@ describe("PhoneInput", () => {
   })
 
   it("does not show a validation message when phone number valid", () => {
-    const tree = renderWithWrappers(
+    const { queryByTestId } = renderWithWrappersTL(
```
Comment (pvinis) [inline src/app/Components/PhoneInput/PhoneInput.tests.tsx:118]: 💜

**Review by pvinis — APPROVED** [review]

**Review by brainbicycle — APPROVED** [review]
Thanks!

**Pull request merged**